client/layout/index.jsx

```diff
@@ -100,6 +101,20 @@ class Layout extends Component {
 		// intentionally don't remove these in unmount
 	}
 
+	renderMasterBar() {
+		if ( 'jetpack-cloud' === config( 'masterbar' ) ) {
+				return <JetpackCloudMasterbar
+					section={ this.props.sectionGroup }
+					isCheckout={ this.props.sectionName === 'checkout' }
+				/>
+		}
+
+		return <MasterbarLoggedIn
```

Comment:
Seems like the logic here could just determine which component to use for the masterbar; and the props are the same. We could do:

```
const MasterbarComponent = 'jetpack-cloud' === config( 'masterbar' ) ? JetpackCloudMasterbar : MasterbarLoggedIn;
return <MasterbarComponent section={ this.props.sectionGroup } isCheckout={ this.props.sectionName === 'checkout' } />;
```

Reply:
Updated.